lambdas/ecs-events-to-slack/notify.py

```diff
@@ -62,6 +68,11 @@ def message_formatter(event, region):
             message_color = "danger"
 
     attachment_fields = [
+        {
+            "title": "Command",
+            "value": command_formatted,
+            "short": "true"
+        },
```

Comment:
From a UX perspective, what do you think about simply not including this field if it's the default?

Reply:
I'm fine with that.  Update incoming.

Reply:
As written you're including the value 2x :-)

Reply:
🤦